openid-sharedsignals-framework-1_0.md

```diff
@@ -1229,6 +1229,12 @@ Errors are signaled with HTTP status codes as follows:
 {: title="Create Stream Errors" #tablecreatestream}
 
 
+##### Validating a Stream Creation Response
+
+* `aud`: the Receiver SHOULD validate the `aud` in the Create Stream Response.
+A Transmitter and Receiver MAY agree upon the audience value out of band.
+Regardless of how the audience value is agreed upon, the Receiver SHOULD ensure that it matches what is expected.
```

Comment:
One nit: Can we say "matches what it expects", rather than "matches what is expected" (clarifies who sets the expectation).